tests/functional/metadata/mongodb/listLifecycleObject/current.spec.js

```diff
@@ -4,7 +4,7 @@ const werelogs = require('werelogs');
 const { MongoMemoryReplSet } = require('mongodb-memory-server');
 const logger = new werelogs.Logger('MongoClientInterface', 'debug', 'debug');
 const MetadataWrapper =
-require('../../../../../lib/storage/metadata/MetadataWrapper');
+    require('../../../../../lib/storage/metadata/MetadataWrapper');
```

Comment:
Yes, the linter (or my IDE?) changed it, it could go into the same line though, as you said. I'll try to do it before merging (I'll wait for other reviews first).

Reply:
Yes, the linter (or my IDE?) changed it, it could go into the same line though, as you said. I'll try to do it before merging (I'll wait for other reviews first).